osbrain/tests/test_agent_async_requests.py

```diff
@@ -2,39 +2,71 @@
 Test file for asynchronous requests.
 """
 import time
+import pytest
```

Comment:
I'd use a blank line to separate built-in packages from external packages (i.e.: between `time` and `pytest`).

This, however, is not really your fault. It should be included in the style/flakes tests. We could add an issue and do it when we find some time. :joy: